pyerf/pyerf.py

```diff
@@ -342,6 +342,43 @@ def erfinv(z):
     # otherwise calculate things.
     return _ndtri((z + 1) / 2.0) / math.sqrt(2)
 
+def erfcinv(z):
+    """
+    Calculate the inverse complementary error function at point ``z``.
+    
+    Parameters
+    ----------
+    z : numeric
+
+    Returns
+    -------
+    float
+
+    References
+    ----------
+    + https://en.wikipedia.org/wiki/Error_function#Inverse_functions
+    + http://functions.wolfram.com/GammaBetaErf/InverseErfc/
+
+    Examples
+    --------
+    >>> round(erfcinv(0.1), 12)
+    1.163087153677
+    >>> round(erfcinv(0.5), 12)
+    0.476936276204    
+    >>> round(erfcinv(0.95), 12)
+    0.04434038791
+    >>> round(erfc(erfcinv(1.9)), 12)
+    -1.163087153677    
+    """
+    if abs(z) > 1:
+        raise ValueError("`z` must be between 0 and 2")
+
+    # Shortcut special cases
+    if z == 0:
```

Comment:
Special cases need to be updated ([source](http://functions.wolfram.com/GammaBetaErf/InverseErfc/03/01/)):
```
erfcinv(0) = inf
erfcinv(1) = 0
```